ios/RNSScreenStackHeaderSubview.mm

```diff
@@ -17,11 +18,13 @@
 namespace react = facebook::react;
 #endif // RCT_NEW_ARCH_ENABLED
 
-@interface RCTBridge (Private)
-+ (RCTBridge *)currentBridge;
-@end
-
 @implementation RNSScreenStackHeaderSubview
+#ifdef RCT_NEW_ARCH_ENABLED
+{
+  react::RNSScreenStackHeaderSubviewShadowNode::ConcreteState::Shared _state;
+  RCTImageLoader *imgLoader;
+}
```

Comment:
You gated almost every part of logic related to it with `NDEBUG` but left a few parts here. Why?

Reply:
+1

Reply:
I needed them since the code with trying to get `imgLoader` was not explicitly surrounded with `NDEBUG` macro till now.  I changed it.